README.md

```diff
@@ -0,0 +1,95 @@
+# CA - Workflow
+
+[![Automated E2E Testing](https://github.com/alexdalene/social-media-client/actions/workflows/e2e-test.yml/badge.svg?event=status)](https://github.com/alexdalene/social-media-client/actions/workflows/e2e-test.yml)
+[![Automated Unit Testing](https://github.com/alexdalene/social-media-client/actions/workflows/unit-test.yml/badge.svg?event=status)](https://github.com/alexdalene/social-media-client/actions/workflows/unit-test.yml)
+
+## Description
+
+Automated testing for an existing project made by @noroff.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Running Tests Locally](#running-tests-locally)
+- [Setting Up Continuous Integration](#setting-up-continuous-integration)
+- [Formatting and linting](#formatting-and-linting)
+
+## Installation
+
+**Clone the Repository:**
+
+```bash
+git clone https://github.com/alexdalene/social-media-client
+cd social-media-client
+```
+
+**Install Dependencies**
+
+```bash
+npm install
+```
+
+**Build SASS**
+
+```bash
+npm run build
+```
+
+## Running tests locally
+
+**Run Cypress**
+
+```bash
+npm run test-e2e
+```
+
+Make sure to change the url for the Cypress tests in cypress/e2e/
+
+```js
+cy.visit('your-url-here');
+```
+
+**Run Jest**
+
+```bash
+npm run test-unit
+```
+
+## Continues Integration
+
+This project uses continuous integration (CI) to automatically run tests on every push. We use GitHub Actions.
+
+## Formatting and Linting
+
+**Format using Prettier**
+
+```bash
+npm run format
+```
+
+**Check linting using ESLint**
+
+```bash
+npm run lint
+```
+
+## Test Results
+
+**Unit Tests**
+
+```bash
+login.test.js - PASSED
+logout.test.js - PASSED
+
+2/2 PASSED
+```
+
+**E2E Tests**
+
+```bash
+errors.cy.js - PASSED
+form.cy.js - FAILED ("Is supposed to fail")
```

Comment:
Det er ikke meningen at tester skal faile. Det er meningen at kode skal kunne faile, men da forventer du en feil i testingen, som igjen gir en passed test.

Du kan sjekke at innlogging fungerer for en bruker som eksisterer med korrekt autorisasjon, men du kan også sjekke for at innlogging ikke fungerer for en bruker som ikke eksisterer. På begge forventer du at testen skal passe, men du sjekker en negativ i test 2.

Dette er veldig viktig å forstå - kontakt meg hvis du vil ha en samtale.

Reply:
Du ønsker alt alltid at alle tester skal passe - og du er ikke fornøyd før alle gjør det.